lib/historic_data_cleaner.rb

```diff
@@ -1,7 +1,7 @@
 # frozen_string_literal: true
 
 class HistoricDataCleaner
-  BATCH_SIZE = 1000
+  BATCH_SIZE = Rails.env.development? || Rails.env.test? ? 5 : 1000
```

Comment:
Personally am not a fan of environment conditional code, I'd sooner use an environment variable here rather than having logic if you need to override it; but in practice I think you can keep the constant of 1000 and then stub the value in your test if you want to see 5 at a time.

Reply:
@dmgarland Thats a great point about stubbing it in the test. I had not thought about that as I didn't write any tests for this.

Thanks, that's a good point about using environment variable vs conditional